Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3469,6 +3471,23 @@ class QuotesWrapper final : public AnimationPropertyWrapperBase {
 };
 DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(QuotesWrapper);
 
+DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(VisibilityWrapper);
```

Comment:
Not your fault but having to do the heap identifier dance for these singletons with uninteresting lifetimes seems bit much.